homeassistant/components/light/hue.py

```diff
@@ -57,6 +57,11 @@ def setup_platform(hass, config, add_devices_callback, discovery_info=None):
 
     if discovery_info is not None:
         host = urlparse(discovery_info[1]).hostname
+        configured_host = _find_host_from_config(hass, filename)
+        if configured_host is not None:
```

Comment:
Please consolidate this `if` with the one below. Also add a comment what is going on here.